app/models/ncr/work_order.rb

```diff
@@ -92,9 +92,13 @@ def client
       "ncr"
     end
 
-    # @todo - this is pretty ugly
     def public_identifier
-      self.cart.id
+      fiscal_year = self.created_at.year
+      if self.created_at.month >= 10
+        fiscal_year += 1
+      end
+      fiscal_year = fiscal_year % 100   # convert to two-digit
+      "FY" + fiscal_year.to_s.rjust(2, "0") + " ##{self.id}"
```

Comment:
Thinking leaving out the space will make it more easily searchable, e.g. `FY15-26`